connection.go

```diff
@@ -463,7 +492,7 @@ func (sc *snowflakeConn) handleMultiExec(ctx context.Context, data execResponseD
 	childResults := getChildResults(data.ResultIDs, data.ResultTypes)
 	for _, child := range childResults {
 		resultPath := fmt.Sprintf("/queries/%s/result", child.id)
```

Comment:
url template `/queries/%s/result` is used at least twice, we can pull it out to a private string constant at the top of this file

Reply:
will do